source/Editor.js

```diff
@@ -1993,11 +1993,49 @@ var keyHandlers = {
             // If the commonAncestor was in a mathjax equation, delete the parent. Namely the mathjax equation.
             if (range.commonAncestorContainer && 
                 range.commonAncestorContainer.parentElement && 
-                checkInSVG.test(range.commonAncestorContainer.nodeName)
+                (checkInSVG.test(range.commonAncestorContainer.nodeName) || checkInSVG.test(range.commonAncestorContainer.parentElement.nodeName))
```

Comment:
Sorry, I just re-read you're earlier comments. Yes, I'm interested if the range.commonAncestorContainer is a child of a Mathjax equation. 

The loop terminates at the body now. You're advocating that the condition be something like:

```
if (mathjaxParent.className === "mathjax''){
     if(mathjaxParent.contains(range.commonAncestorContainer)){
           // remove the mathjaxParent.
     }
 break;
}
```